admin-part/src/main/kotlin/Scope.kt

```diff
@@ -60,8 +60,18 @@ class ActiveScope(
 
     val activeSessions = AtomicCache<String, ActiveSession>()
 
-    val methodsCoveredByTestCache = getCache<Int, MethodsCoveredByTest>(realtimeCalculationCache)
-
+    /**
+     * The cache is used for real-time coverage calculations.
+     * Enabling and disabling depends on the system property: "drill.plugins.test2code.features.realtime.cache".
+     * Only tests in the finalize state are added to the cache
+     */
+    val methodsCoveredByTestCache = getCache<TypedTest, MethodsCoveredByTest>(realtimeCalculationCache)
+
+    /**
+     * The cache is used for real-time coverage calculations.
+     * Enabling and disabling depends on the system property: "drill.plugins.test2code.features.realtime.cache".
+     * Only tests in the finalize state are added to the cache
+     */
     val bundlesByTestsCache = getCache<TypedTest, BundleCounter>(realtimeCalculationCache)
```

Comment:
smth strange in naming: 
- bundle**S**ByTest**S**Cache 
- methodsCoveredByTe**ST**Cache